pilot/cmd/pilot-agent/status/server.go

```diff
@@ -421,7 +425,7 @@ func (s *Server) Run(ctx context.Context) {
 }
 
 func (s *Server) handlePprofIndex(w http.ResponseWriter, r *http.Request) {
-	if !isRequestFromLocalhost(r) {
+	if !istioNetUtil.IsRequestFromLocalhost(r) {
```

Comment:
All of these should probably be a middleware too but it was like this before.

Reply:
Yes, I did not modify existing handlers.